src/Compilers/CSharp/Test/Semantic/CSharpCompilerSemanticTest.csproj

```diff
@@ -47,8 +55,10 @@
       <Name>PdbUtilities</Name>
     </ProjectReference>
   </ItemGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' "></PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' "></PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+  </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|ARM' ">
```

Comment:
@Pilchie what specifically does the IDE need for these groups?  Is there something I can look at to see what the restrictions are?  

The ARM entries seem 100% unnecessary, would be surprised if the IDE cared. Or maybe not surprised just dissapointed 😦